### DISCUSSION TITLE
TerserPlugin, 'mangle' work incorrectly

### DISCUSSION THREAD

**Discussion by Krooshk** [webpack/webpack#18402] [Q&A]
I want to mangle all names of functions and classes and also methods of classes.

For this i use following in webpack : 
`... minimizer: [
                new TerserPlugin({
                    terserOptions: {
                        mangle: {
                            properties: true
                        }
                    }
                })
               ]
...`

All code is mangled, but something went wrong.
I get such error:  

> Uncaught TypeError: t.Tge is not a function

In this place, where create instance and export:
`export const example= new Example();`

I suppose it's lost something.

**Comment by alexander-akait** [comment]
Please report it to `terser` repo, we can't fix it here, I can help with it if you provide reproducible test repo, thank you
